kogito-editors-js/packages/feel-input-component/showcase/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
I would add a new line at the end of file

Reply:
Fixed as well! Thanks 🚀